packages/react-hooks/CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 - Added `useDebouncedValue` hook ([#1354](https://github.com/Shopify/quilt/pull/1354))
 
+### Fixed
+
+- Removed `window` for `[set|clear]Timeout` from `useDebouncedValue` hook ([#1360](https://github.com/Shopify/quilt/pull/1360))
```

Comment:
Where'd the `Added useDebouncedValue hook` changelog item go? Does this branch need a rebase?

Reply:
It is right above https://github.com/Shopify/quilt/pull/1360/files#diff-6c41914fef434be5eed762a9c9d10c5fR12.

Should I merge these two into the single `Added`? @tleunen ?

Reply:
Since it's the same version, I'd remove the log about fixing it